### ISSUE TITLE
Error when trying to update sensors when device isn't testing

### ISSUE THREAD

**Issue by Intecpsp** [ydogandjiev/hass-sutro#127]
My replacement cartridge is late due to some shipping issues on Sutro's end, so my device has stopped testing. 

I can manually reload the integration and get the latest temp, but the other sensors are failing because they're getting back null. 

I'd like to at least keep the temp going in these cases. Can some safety null checks be put in place to allow things to update when others are having issues?

```
Logger: homeassistant.components.sensor
Source: helpers/entity_platform.py:600
integration: Sensor (documentation, issues)
First occurred: June 10, 2024 at 2:59:19 PM (2 occurrences)
Last logged: 7:28:44 AM

Error adding entity sensor.sutro_alkalinity_sensor for domain sensor with platform sutro
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 600, in _async_add_entities
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 914, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1355, in add_to_platform_finish
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1000, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1126, in _async_write_ha_state
    state, attr, capabilities, shadowed_attr = self.__async_calculate_state()
                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1061, in __async_calculate_state
    state = self._stringify_state(available)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1006, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 543, in state
    value = self.native_value
            ^^^^^^^^^^^^^^^^^
  File "/config/custom_components/sutro/sensor.py", line 112, in native_value
    return float(self.coordinator.data["me"]["pool"]["latestReading"]["alkalinity"])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: float() argument must be a string or a real number, not 'NoneType'
```

**Comment by Intecpsp** [comment]
Another very similar one

```
This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/sutro/sensor.py:112
integration: Sutro (documentation, issues)
First occurred: June 10, 2024 at 3:29:19 PM (33 occurrences)
Last logged: 7:58:44 AM

Error doing job: Task exception was never retrieved (None)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 255, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 411, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 165, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 491, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1000, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1126, in _async_write_ha_state
    state, attr, capabilities, shadowed_attr = self.__async_calculate_state()
                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1061, in __async_calculate_state
    state = self._stringify_state(available)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1006, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 543, in state
    value = self.native_value
            ^^^^^^^^^^^^^^^^^
  File "/config/custom_components/sutro/sensor.py", line 112, in native_value
    return float(self.coordinator.data["me"]["pool"]["latestReading"]["alkalinity"])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: float() argument must be a string or a real number, not 'NoneType'
```

**Comment by ydogandjiev** [comment]
Addressed in v1.0.0-beta.0:
https://github.com/ydogandjiev/hass-sutro/releases/tag/v1.0.0-beta.0

@Intecpsp, please consider updating to v1.0.0-beta.0 and confirming that you no longer see the issue and everyhing works as expected.
